google-cloud-bigquerystorage/clirr-ignored-differences.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- see http://www.mojohaus.org/clirr-maven-plugin/examples/ignored-differences.html -->
+<differences>
+  <difference>
+    <differenceType>4001</differenceType>
+    <className>com/google/cloud/bigquery/storage/v1beta1/Storage</className>
```

Comment:
[ERROR] 4001: com.google.cloud.bigquery.storage.v1beta1.Storage$ReadRowsResponse$RowsCase: Removed com.google.protobuf.AbstractMessageLite$InternalOneOfEnum from the set of implemented interfaces
[ERROR] 4001: com.google.cloud.bigquery.storage.v1beta1.Storage$ReadSession$SchemaCase: Removed com.google.protobuf.AbstractMessageLite$InternalOneOfEnum from the set of implemented interfaces

I'm guessing you need to include the inner class name here

Reply:
this is still pending

```xml
<differences>
  <difference>
    <className>com/google/cloud/bigquery/storage/v1beta1/BigQueryStorageGrpc</className>
    <field>*</field>
    <differenceType>6001</differenceType>
  </difference>
</differences>
```